.github/workflows/release.yaml

```diff
@@ -0,0 +1,42 @@
+name: "Release"
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseBranch:
+        description: "Release branch name"
+        required: true
+        type: string
+      oldVersion:
```

Comment:
This could also be found by looking at the existing tags (not a blocker)

Reply:
Done